generators/cleanup.js

```diff
@@ -221,6 +221,7 @@ function cleanupOldServerFiles(generator, javaDir, testDir, mainResourceDir, tes
         generator.removeFile(`${testDir}web/rest/ClientForwardControllerIT.java`);
     }
     if (generator.isJhipsterVersionLessThan('6.6.1')) {
+        generator.removeFile(`${javaDir}security/oauth2/JwtAuthorityExtractor.java`);
```

Comment:
Should be after:
if (generator.isJhipsterVersionLessThan('6.8.0')) {

Reply:
@Tcharl Can you make this minor change?

The file should be removed only on version 6.8.0.

Reply:
I fixed it. So it can be merged if the CI pass